java/datastore/src/test/java/com/google/datastore/v1/client/RemoteRpcTest.java

```diff
@@ -122,54 +119,48 @@ public void testPlainTextException() {
 
   @Test
   public void testGzipHack_NonGzip() throws Exception {
```

Comment:
I think we can probably just remove this test.

Reply:
you're right, done